configs/opencv-ovms/gst_capi/pipelines/face_detection/main.cpp

```diff
@@ -495,14 +496,8 @@ std::string getVideoPipelineText(std::string mediaPath, ObjectDetectionInterface
         modelFrameShape = textDet->getModelInputShape();
     }
 
-    int frame_width = modelFrameShape[1];
-    int frame_height = modelFrameShape[2];
-
-    if (_render)
-    {
-        frame_width = _window_width;
-        frame_height = _window_height;
-    }
+    int frame_width = _window_width;
+    int frame_height = _window_height;
```

Comment:
what is the logic change here for?

Reply:
we should revert this. The current logic is hard coded to 1280x720 where the older logic would get the sized based on the model requirements.

Reply:
We can open a different issue to handle this logic in the future.

Reply:
unfortunately, " get the sized based on the model requirements" was also hard coded, yeah, it makes sense to handle it in the future